daemons/gptp/common/ptp_message.cpp

```diff
@@ -1502,12 +1510,23 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	    Timestamp prev_peer_ts_theirs;
 	    FrequencyRatio rate_offset;
 	    if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+		    FrequencyRatio upper_limit, lower_limit;
+		    upper_limit = UPPER_FREQ_LIMIT*2;
+		    upper_limit /= US_PER_SEC;
+		    upper_limit += 1;
+		    lower_limit = UPPER_FREQ_LIMIT*2;
```

Comment:
Shouldn't the lower_limit be based on LOWER_FREQ_LIMIT? I am also a little unclear on why we're multiplying by 2 and then adding 1 to each of these.  Would be nice to have a comment explaining what the math is for and where it came from.